README.md

```diff
@@ -91,113 +94,223 @@ The front end of the integration is now complete.
 
 ### Create a Scenario
 
-You will now need to your SDK ID, Scenario ID and call back URL ready from your application dashboard. 
+You will now need your SDK ID, Scenario ID and call back URL ready from your application dashboard. 
 
-The SDK will need to be initialised, please and add the below scenario method:
+The SDK will need to be initialised, please add like below, your scenarios. Note that the SDK can support now multiple scenarios:
```

Comment:
I would replace that whole sentence as "For each of the scenario you want to handle, would will need to add them to the YotiSDK like below".

Reply:
done